changelogs/unreleased/gh-4675-move-info-messages-to-info-logging-level.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua/log
+
+* Some information messages have been moved from the critical
+to the info logging level. (gh-4675)
```

Comment:
I think we should also use the info level when writing a checkpoint:

https://github.com/tarantool/tarantool/blob/c463e56b90bc8ed21be6161a62df759924daf83c/src/box/memtx_engine.cc#L641

Reply:
Fixed